### PR TITLE
Labels script

### PR DESCRIPTION
**Pull request by misirov**

### CODE REVIEW

File: create-labels.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        "ff0000",
+        "B60205",
+        "D93F0B",
+        "FBCA04",
+        "1D76DB",
+        "B4E197",
+        "5319E7",
+        "0E8A16",
+        "bfdadc"
+    ]
```
Comment (hrkrshnn) [inline create-labels.py]: I'd probably use a dictionary.  Something like
```python
labels = [
    {'severity': 'Severity: Critical Risk', 'color': 'ff00000'},
    {'severity': ...}
```

File: create-labels.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    i = 0
+    if len(new_labels) == len(colors):
+        for n in new_labels:
+            data={
+                "name":n,
+                "color":colors[i]
+            }
+            print(data)
+            r = requests.post(f"https://api.github.com/repos/{username}/{repo}/labels", headers=headers, json=data)
```
Comment (hrkrshnn) [inline create-labels.py]: Any reason you went with a raw request rather than using pygithub?

File: create-labels.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+# __ MAIN __
+
+print("Hello! This script will first delete all labels from the target repository and create custom ones. Please enter the following details:\n")
+while(True):
+    token = input(str("1) Access token: "))
+    username = input(str("2) User / Organization name: "))
+    repo = input(str("3) Target repo name: "))
+    if token != "" and repo != "" and username != "":
+        break
+    print("Please fill in the details.")
```
Comment (hrkrshnn) [inline create-labels.py:61]: Maybe better to get it from cli?

File: create-labels.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+create_new_labels()
+
+print("Please confirm old labels have been deleted and new ones have been created.")
+
+
+
+
+
+
+
```
Comment (hrkrshnn) [inline create-labels.py]: Extra space.

File: create-labels.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    token = input(str("1) Access token: "))
+    username = input(str("2) User / Organization name: "))
+    repo = input(str("3) Target repo name: "))
+    if token != "" and repo != "" and username != "":
+        break
+    print("Please fill in the details.")
+ 
+
+headers = {
+    "Authorization":f"token {token}",
```
Comment (hrkrshnn) [inline create-labels.py]:
```suggestion
    "Authorization": f"token {token}",
```
Nitpick

File: create-labels.py [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+    "Accept": "application/vnd.github.v3+json",
+    'User-Agent': 'request'
```
Comment (hrkrshnn) [inline create-labels.py]: Nitpick. Uses `'` and `"`. Maybe just pick one and be consistent?

File: create-labels.py [hunk 7]
```diff
@@ -0,0 +1,100 @@
+# This script removes default github labels and adds custom ones using python requests and the github API
+# To work it needs: 1) User token, 2) Target username / organization name and target repository name and the 
+# By Pablo and Spearbit with <3
+# License: MIT
```
Comment (hrkrshnn) [inline create-labels.py]: Maybe this can be a python docstring?

**Review by hrkrshnn — COMMENTED** [review]
Generally, looks good. Left some minor comments.

File: create-labels.py (hunk 1 above)
Comment (misirov) [reply create-labels.py]: Makes sense, i'll address it

File: create-labels.py (hunk 2 above)
Comment (misirov) [reply create-labels.py]: Did not think that using a wrapper was necessary for such trivial task, it makes the script lighter and easier to modify for anyone

File: create-labels.py (hunk 5 above)
Comment (misirov) [reply create-labels.py]: fair enough

**Pull request merged**